packages/fx-core/resource/package.nls.json

```diff
@@ -1150,6 +1150,10 @@
   "driver.file.updateJson.description": "Add or update appsettings to JSON file.",
   "driver.file.updateJson.summary.withTarget": "The value for appsettings has been generated successfully to %s.",
   "driver.env.error.folderNotExist": "Folder %s does not exist.",
+  "driver.deploy.FailedListPublishingCredentialsRemoteError": "Failed to list publishing credentials due to remote service error. \nPlease check your account and try again later.",
```

Comment:
"later" is vague, either remove it, or make it more specific.

Reply:
Maybe you can use "try it again" instead of "try again later", so users can know what to do and expect.